File: FLM_SubconLabelSystem/Library/WIP_Process/Equipment_RFID_Listing.vb
Imports System
Imports System.Diagnostics

Namespace WIP_Process
	Public Class Equipment_RFID_Listing
		Private str_Equipment_RFID As String

		Public Property Equipment_RFID As String
			Get
				Return Me.str_Equipment_RFID
			End Get
			Set(ByVal value As String)
				Me.str_Equipment_RFID = value
			End Set
		End Property

		<DebuggerNonUserCode>
		Public Sub New()
			MyBase.New()
		End Sub
	End Class
End Namespace

File: FLM_SubconLabelSystem/Library/WIP_Process/Equipment_RFID_Listing.cs
using System;
using System.Diagnostics;

namespace WIP_Process
{
    public class Equipment_RFID_Listing
    {
        private string str_Equipment_RFID;

        public string Equipment_RFID
        {
            get
            {
                return this.str_Equipment_RFID;
            }
            set
            {
                this.str_Equipment_RFID = value;
            }
        }

        [DebuggerNonUserCode]
        public Equipment_RFID_Listing() : base()
        {
        }
    }
}
